File: app/mobile/modules/platform-capabilities/ios/PlatformCapabilitiesModule.swift
import ExpoModulesCore

// Keep in lockstep with android/.../PlatformCapabilitiesModule.kt.
private let CAPABILITY_PLATFORM_VERSION: Int = 1
private let CAPABILITIES: [String] = []

public class PlatformCapabilitiesModule: Module {
  public func definition() -> ModuleDefinition {
    Name("PlatformCapabilities")

    Constants([
      "capabilityPlatformVersion": CAPABILITY_PLATFORM_VERSION,
      "capabilities": CAPABILITIES,
    ])
  }
}

File: app/mobile/modules/platform-capabilities/android/src/main/java/expo/modules/platformcapabilities/PlatformCapabilitiesModule.kt
package expo.modules.platformcapabilities

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

// Keep in lockstep with ios/PlatformCapabilitiesModule.swift.
private const val CAPABILITY_PLATFORM_VERSION: Int = 1
private val CAPABILITIES: List<String> = emptyList()

class PlatformCapabilitiesModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("PlatformCapabilities")

    Constants(
      "capabilityPlatformVersion" to CAPABILITY_PLATFORM_VERSION,
      "capabilities" to CAPABILITIES,
    )
  }
}
